terraform/environment/terraform.tfvars.json

```diff
@@ -334,6 +379,15 @@
           "arn:aws:execute-api:eu-west-2:288342028542:*/*/GET/health"
         ]
       },
+      "lpa_store_service": {
+        "base_url": "https://lpa-store.api.opg.service.justice.gov.uk",
+        "api_arns": [
+          "arn:aws:execute-api:eu-west-1:764856231715:*/*/PUT/lpas/*",
+          "arn:aws:execute-api:eu-west-2:764856231715:*/*/PUT/lpas/*",
+          "arn:aws:execute-api:eu-west-1:764856231715:*/*/GET/health-check",
+          "arn:aws:execute-api:eu-west-2:764856231715:*/*/GET/health-check"
+        ]
+      },
```

Comment:
Assuming this is fine but thought I should check, is the LPA store being used in any live environments yet?

Reply:
it is deployed to production, it doesn't do anything though